dashboard/api/pkg/model/moderation/types.go

```diff
@@ -0,0 +1,23 @@
+package moderation
+
+import (
+	"context"
+
+	"github.com/bacalhau-project/bacalhau/dashboard/api/pkg/types"
+	"github.com/bacalhau-project/bacalhau/pkg/bidstrategy"
+	"github.com/bacalhau-project/bacalhau/pkg/verifier"
+)
+
+type ExecutionModerator interface {
+	ShouldExecute(context.Context, *bidstrategy.JobSelectionPolicyProbeData) (*bidstrategy.BidStrategyResponse, error)
+}
+
+type ResultsModerator interface {
+	ShouldVerify(context.Context, verifier.VerifierRequest) ([]verifier.VerifierResult, error)
```

Comment:
A minor nitpick I think, but when I read `ShouldVerify` I'm not sure I would expect it to return a `[]verifier.VerifierResult`, in my mind it's either a command (setting a value) or querying for a bool response. Presumably with this (and ShouldExecute) it's up to the caller to make sense of the response?

Reply:
Yeah, I suppose it's really "which should be verified" – and those ones are returned. `ShouldExecute` is more obviously a boolean – well, it contains a tri-state value along with a human-readable reason. I think I prefer the symmetry and also can't think of any better names... suggestions welcomed

Reply:
Well, I suppose this method is the one doing the verification – so any returned results have been verified one way or the other by this method. So perhaps we should just follow the verifier interface and use `Verify`.